cmd/run.go

```diff
@@ -110,6 +111,10 @@ A full workflow is capable of running the following steps:
 			Description: "Run using the current CLI version instead of the version specified in the workflow file",
 			Hidden:      true,
 		},
+		flag.StringSliceFlag{
+			Name:        "bundle-tag",
```

Comment:
should this not be plural? Also didn't we decide on `registry-tags`?

Reply:
Hmm is it better to plural or not. Should be registry-tag I think